Hubo un problema con los datos ingresados
Hubo un problema con los datos ingresados
Hubo un problema con los datos ingresados
Hubo un problema con los datos ingresados
Hubo un problema con los datos ingresados
System.InvalidOperationException: Error en el documento XML (0, 0). ---> System.IO.FileNotFoundException: No se pudo encontrar el archivo 'C:\Users\Alvaro\Desktop\tp_laboratorio_2\TP3\Elena.Alvaro.2D.TPFinal\TP_03\bin\Debug\Material - Leche.xml'.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   en System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize)
   en System.Xml.XmlDownloadManager.GetStream(Uri uri, ICredentials credentials, IWebProxy proxy, RequestCachePolicy cachePolicy)
   en System.Xml.XmlUrlResolver.GetEntity(Uri absoluteUri, String role, Type ofObjectToReturn)
   en System.Xml.XmlTextReaderImpl.OpenUrlDelegate(Object xmlResolver)
   en System.Threading.CompressedStack.runTryCode(Object userData)
   en System.Runtime.CompilerServices.RuntimeHelpers.ExecuteCodeWithGuaranteedCleanup(TryCode code, CleanupCode backoutCode, Object userData)
   en System.Threading.CompressedStack.Run(CompressedStack compressedStack, ContextCallback callback, Object state)
   en System.Xml.XmlTextReaderImpl.OpenUrl()
   en System.Xml.XmlTextReaderImpl.Read()
   en System.Xml.XmlTextReader.Read()
   en System.Xml.XmlReader.MoveToContent()
   en Microsoft.Xml.Serialization.GeneratedAssembly.XmlSerializationReaderMateriales.Read3_Materiales()
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Xml.Serialization.XmlSerializer.Deserialize(XmlReader xmlReader, String encodingStyle, XmlDeserializationEvents events)
   en System.Xml.Serialization.XmlSerializer.Deserialize(XmlReader xmlReader)
   en Serializadora.Xml`1.Leer(String ruta, T& datos) en C:\Users\Alvaro\Desktop\tp_laboratorio_2\TP3\Elena.Alvaro.2D.TPFinal\Serializadora\Xml.cs:línea 49
Entidades.MisExcepciones: Error en el documento XML (0, 0).
   en Serializadora.Xml`1.Leer(String ruta, T& datos) en C:\Users\Alvaro\Desktop\tp_laboratorio_2\TP3\Elena.Alvaro.2D.TPFinal\Serializadora\Xml.cs:línea 56
   en Serializadora.Serializadora`1.Leer(String ruta) en C:\Users\Alvaro\Desktop\tp_laboratorio_2\TP3\Elena.Alvaro.2D.TPFinal\Serializadora\Serializadora.cs:línea 38
System.InvalidOperationException: Error en el documento XML (0, 0). ---> System.IO.FileNotFoundException: No se pudo encontrar el archivo 'C:\Users\Alvaro\Desktop\tp_laboratorio_2\TP3\Elena.Alvaro.2D.TPFinal\TP_03\bin\Debug\Material - Leche.xml'.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   en System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize)
   en System.Xml.XmlDownloadManager.GetStream(Uri uri, ICredentials credentials, IWebProxy proxy, RequestCachePolicy cachePolicy)
   en System.Xml.XmlUrlResolver.GetEntity(Uri absoluteUri, String role, Type ofObjectToReturn)
   en System.Xml.XmlTextReaderImpl.OpenUrlDelegate(Object xmlResolver)
   en System.Threading.CompressedStack.runTryCode(Object userData)
   en System.Runtime.CompilerServices.RuntimeHelpers.ExecuteCodeWithGuaranteedCleanup(TryCode code, CleanupCode backoutCode, Object userData)
   en System.Threading.CompressedStack.Run(CompressedStack compressedStack, ContextCallback callback, Object state)
   en System.Xml.XmlTextReaderImpl.OpenUrl()
   en System.Xml.XmlTextReaderImpl.Read()
   en System.Xml.XmlTextReader.Read()
   en System.Xml.XmlReader.MoveToContent()
   en Microsoft.Xml.Serialization.GeneratedAssembly.XmlSerializationReaderMateriales.Read3_Materiales()
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Xml.Serialization.XmlSerializer.Deserialize(XmlReader xmlReader, String encodingStyle, XmlDeserializationEvents events)
   en System.Xml.Serialization.XmlSerializer.Deserialize(XmlReader xmlReader)
   en Serializadora.Xml`1.Leer(String ruta, T& datos) en C:\Users\Alvaro\Desktop\tp_laboratorio_2\TP3\Elena.Alvaro.2D.TPFinal\Serializadora\Xml.cs:línea 49
Entidades.MisExcepciones: Error en el documento XML (0, 0).
   en Serializadora.Xml`1.Leer(String ruta, T& datos) en C:\Users\Alvaro\Desktop\tp_laboratorio_2\TP3\Elena.Alvaro.2D.TPFinal\Serializadora\Xml.cs:línea 56
   en Serializadora.Serializadora`1.Leer(String ruta) en C:\Users\Alvaro\Desktop\tp_laboratorio_2\TP3\Elena.Alvaro.2D.TPFinal\Serializadora\Serializadora.cs:línea 38
Hubo un problema con actualizar la lista
System.InvalidOperationException: Error en el documento XML (0, 0). ---> System.IO.FileNotFoundException: No se pudo encontrar el archivo 'C:\Users\Alvaro\Desktop\tp_laboratorio_2\TP3\Elena.Alvaro.2D.TPFinal\TP_03\bin\Debug\Material - Leche.xml'.
   en System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   en System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   en System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize)
   en System.Xml.XmlDownloadManager.GetStream(Uri uri, ICredentials credentials, IWebProxy proxy, RequestCachePolicy cachePolicy)
   en System.Xml.XmlUrlResolver.GetEntity(Uri absoluteUri, String role, Type ofObjectToReturn)
   en System.Xml.XmlTextReaderImpl.OpenUrlDelegate(Object xmlResolver)
   en System.Threading.CompressedStack.runTryCode(Object userData)
   en System.Runtime.CompilerServices.RuntimeHelpers.ExecuteCodeWithGuaranteedCleanup(TryCode code, CleanupCode backoutCode, Object userData)
   en System.Threading.CompressedStack.Run(CompressedStack compressedStack, ContextCallback callback, Object state)
   en System.Xml.XmlTextReaderImpl.OpenUrl()
   en System.Xml.XmlTextReaderImpl.Read()
   en System.Xml.XmlTextReader.Read()
   en System.Xml.XmlReader.MoveToContent()
   en Microsoft.Xml.Serialization.GeneratedAssembly.XmlSerializationReaderMateriales.Read3_Materiales()
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Xml.Serialization.XmlSerializer.Deserialize(XmlReader xmlReader, String encodingStyle, XmlDeserializationEvents events)
   en System.Xml.Serialization.XmlSerializer.Deserialize(XmlReader xmlReader)
   en Serializadora.Xml`1.Leer(String ruta, T& datos) en C:\Users\Alvaro\Desktop\tp_laboratorio_2\TP3\Elena.Alvaro.2D.TPFinal\Serializadora\Xml.cs:línea 49
Entidades.MisExcepciones: Error en el documento XML (0, 0).
   en Serializadora.Xml`1.Leer(String ruta, T& datos) en C:\Users\Alvaro\Desktop\tp_laboratorio_2\TP3\Elena.Alvaro.2D.TPFinal\Serializadora\Xml.cs:línea 56
   en Serializadora.Serializadora`1.Leer(String ruta) en C:\Users\Alvaro\Desktop\tp_laboratorio_2\TP3\Elena.Alvaro.2D.TPFinal\Serializadora\Serializadora.cs:línea 38
Hubo un problema con actualizar la lista
Hubo un problema con los datos ingresados
Hubo un problema con los datos ingresados
